test/ng/intervalSpec.js

```diff
@@ -142,6 +142,23 @@ describe('$interval', function() {
   }));
 
 
+  it('should allow you to specify a number of arguments', inject(function($interval, $window) {
+    var task1 = jasmine.createSpy('task1'),
+        task2 = jasmine.createSpy('task2'),
+        task3 = jasmine.createSpy('task3');
+    $interval(task1, 1000, 1, true, 'Task1');
+    $interval(task2, 1000, 1, true, 'Task2');
+    $interval(task3, 1000, 1, true, 'I', 'am', 'a', 'Task3', 'spy');
+
+    $window.flush(1000);
+    expect(task1).toHaveBeenCalledWith('Task1');
+    $window.flush(1000);
+    expect(task2).toHaveBeenCalledWith('Task2');
+    $window.flush(1000);
+    expect(task3).toHaveBeenCalledWith('I', 'am', 'a', 'Task3', 'spy');
+  }));
```

Comment:
I updated the tests (and rebase master) with an `$interval` count of 2 otherwise the second part of this won't work since the `.reset()` only resets the state of the spy and not `$interval`.